R/population-workflow.R

```diff
@@ -34,6 +34,7 @@ PopulationWorkflow <- R6::R6Class(
       super$initialize(...)
 
       # TO DO: include task parameters from initialization ?
+      self$resetReportSettings()
       self$simulatePopulationSettings()

```

Comment:
This has nothing to do with this PR but I am having trouble with the naming

For example simulatePopulationSettings does not `simulate` anything. Rather it create a task that will later be run correct? Should not we call these methods something different so that the non expert reader of the code does not believe that the simulation is going to be run right away?

/cc @pchelle @abdullahhamadeh

Reply:
I'd like feedback on this even though I am going to merge. 
@abdullahhamadeh

Reply:
we should create an issue for this because the mean model simulation task has a similar naming

Reply:
Sure. But do you concur that the naming is misleading OR is it me who does not understand the purpose of the functions?

Reply:
I agree, I think they should be simulationTaskSettings  and populationSimulationTaskSettings

Reply:
Also the task should be simulationTask and populationSimulationTask

Reply:
I'd even go as far as saying it should be something like `create` or `init`
for example createSimulationTask or sthg like this